pkg/api/v1/schema.go

```diff
@@ -810,6 +810,10 @@ type DHCPOptions struct {
 	// If specified will pass option 66 to interface's DHCP server
 	// +optional
 	TFTPServerName string `json:"tftpServerName,omitempty"`
+
```

Comment:
nit: remove this empty line

Reply:
thanks @SchSeba , removed this vertical space.